tasks/volume.yml

```diff
@@ -4,7 +4,7 @@
       {{
       dict(
         mountpoint=(appuio_openshift_localstorage_mountpoint_parent ~ '/' ~ item.key),
-        mountopts="defaults",
+        mountopts=(appuio_openshift_localstorage_mountopts | join(',')),
```

Comment:
Please retain a default of `defaults` in case a user unsets `appuio_openshift_localstorage_mountopts`: `appuio_openshift_localstorage_mountopts | default([]) | join(',') | default('defaults')` (or similar).

Reply:
Instead of defaulting to `['defaults']`, I opted to always prepend `defaults` to the list of mount options/